queries/goctl/folds.scm

```diff
@@ -0,0 +1,26 @@
+; Fold rules for goctl
+; Fold types
+(typeGroupSpec) @fold
+
+(typeSingleSpec) @fold
+
+(typeStruct) @fold
+
+; Fold services
+(serviceSpec) @fold
+
+(serviceBody) @fold
+
+; Fold import groups
+(importGroup) @fold
+
+(importSpec) @fold
```

Comment:
I think this one should be `(importStatement)` (consecutive imports)

Reply:
all changes done.
btw, I had to set the following configuration in neovim for goctl to take effect,but this doesn't look very convenient, can it be set by default in treesitter.
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/51357674/89af137b-d7fd-412a-87e3-61d525131497)

Reply:
Again, no. Filetype detection is out of scope here, and in any case `api` should not be the filetype, it's much too generic.